app/Http/Requests/UpdatePasswordRequest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdatePasswordRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        
+        return [
+            'current_password' => ['required', 'string'],
+            'new_password' => ['required', 'string', 'min:8', 'confirmed'],
+            'confirm_new_password' => ['required', 'string', 'same:new_password'],
+        ];
```

Comment:
You can remove `confirm_new_password `since you added `confirmed `for `new_password`. For confirmation via Postman, you can include new_password_confirmation in the request body.